feature-libs/product-configurator/rulebased/components/attribute/types/multi-selection-image/configurator-attribute-multi-selection-image.component.ts

```diff
@@ -92,14 +104,30 @@ export class ConfiguratorAttributeMultiSelectionImageComponent
     );
   }
 
+  /**
+   * Extract corresponding value price formula parameters.
+   * For the multi-selection attribute types the complete price formula should be displayed at the value level.
+   *
+   * @param {Configurator.Value} value - Configurator value
```

Comment:
According to SPA code guidelines it is not recommended to add a type of the parameter.
Could you, please, remove {xxx}?

Reply:
will remove the comment as i did not intended to add it in the first place.